pages/collections3/[collectionAddress]/about.tsx

```diff
@@ -0,0 +1,59 @@
+import { CollectionRaisedCard } from '@/common/components/collections/CollectionRaisedCard';
+import { CollectionPage } from '@/common/components/collections/collections.util';
+import { FollowItem } from '@/common/components/elements/FollowModal';
+import CollectionLayout from '@/layouts/CollectionLayout';
+import { GetServerSideProps } from 'next';
+import React, { ReactElement } from 'react';
+import { graphqlRequestClient } from 'src/graphql/graphql-request';
+import { GetCollectionQuery } from 'src/graphql/indexerTypes';
+import { getSdk } from 'src/graphql/indexerTypes.ssr';
+
+export const getServerSideProps: GetServerSideProps<CollectionPage> = async (context) => {
+  const collectionAddress = (context.query.collectionAddress || '') as string;
+
+  const { getCollection } = getSdk(graphqlRequestClient);
+
+  const collection = await getCollection({
+    address: collectionAddress,
+  });
+
+  return {
+    props: {
+      collectionAddress,
+      collection: collection.nft,
+    },
+  };
+};
+
+export default function CollectionAboutPage(props: CollectionPage) {
+  return (
+    <div className="mt-20 space-y-20">
+      <CollectionRaisedCard>
+        <h2 className="text-2xl font-semibold">About this collection</h2>
+        <p>{props.collection?.description}</p>
+      </CollectionRaisedCard>
+      <CollectionRaisedCard>
+        <h2 className="text-2xl font-semibold">Creators of this collection</h2>
+        <div className="mt-10 space-y-10">
+          {props.collection?.creators.map((cc) => (
+            <FollowItem
+              key={cc.address}
+              source="collectionCreators"
+              user={{
+                address: cc.address,
+                profile: {
+                  handle: cc.profile?.handle,
+                  profileImageUrl: cc.profile?.profileImageUrlLowres,
+                } as any,
+              }}
+            />
+          ))}
+        </div>
+      </CollectionRaisedCard>
+    </div>
+  );
+}
+
+CollectionAboutPage.getLayout = function getLayout(page: ReactElement) {
```

Comment:
I recommend doing the layout as a component like in marketplace so it has access to the app context and can get server side props.

https://github.com/holaplex/marketplace/blob/dev/src/pages/_app.tsx#L81

Reply:
This is clever!

Reply:
@kristianeboe Copying marketplaces exactly made typescript unhappy. I have made updates in [my branch](https://github.com/holaplex/holaplex/compare/austin/348/discover?expand=1) that seem to do the trick (see _app.tsx)